<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationClass</class>
 <widget class="QMainWindow" name="SimulationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>仿真系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/FOD/Resources/logo.png</normaloff>:/FOD/Resources/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="picToolBar">
   <property name="windowTitle">
    <string>仿真系统</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_Pic"/>
   <addaction name="open_List"/>
   <addaction name="load_Pic"/>
   <addaction name="modelList"/>
   <addaction name="result"/>
  </widget>
  <action name="open_Pic">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="open_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>处理列表</string>
   </property>
  </action>
  <action name="modelList">
   <property name="text">
    <string>模型列表</string>
   </property>
  </action>
  <action name="result">
   <property name="text">
    <string>检测结果</string>
   </property>
  </action>
  <action name="load_Pic">
   <property name="text">
    <string>批量加载</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
